CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @rancher/rancher-team-1-neo-dev
```

Comment:
I'm not sure if we should put team1 as the owner here. Per the Slack conversation, because this is a cross team image.

Reply:
I'm fine with adding 3 teams as long as they all own it.

Reply:
It's difficult to say if all 3 teams will review and own it, but it's currently a shared image.